### PR TITLE
Add -inf inf and nan

### LINKED ISSUE #8650: Document usable values in reclassify raster tables
## Description
Values that can be used in reclassify tables are not documented (that I can find). 
So far I have found in addition to numbers **-inf**, **inf**, and **nan** can also be used. Before I take a crack at putting this in the documentation are there others or is there a place that already lists valid inputs.  Thanks!

Page URL:
https://docs.qgis.org/testing/en/docs/user_manual/processing_algs/qgis/rasteranalysis.html?highlight=reclassify#reclassify-by-table

### PR DESCRIPTION
**Pull request by baswein**
Goal: Document other values that can be used in reclassify by table. 

Ticket(s): fix #8650

- [ x] Backport to LTR documentation is requested

### CODE REVIEW

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst [hunk 1]
```diff
@@ -2869,16 +2869,18 @@ Basic parameters
      - ``MIN_FIELD``
      - [tablefield: numeric]
      - Field with the minimum value of the range for the class.
+       Use -inf to capture the lowest possible value.
    * - **Maximum class value field**
      - ``MAX_FIELD``
      - [tablefield: numeric]
      - Field with the maximum value of the range for the class.
+       Use inf to capture the highest possible value.
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/rasteranalysis.rst:2877]:
```suggestion
       Use ``inf`` to capture the highest possible value.
```

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst [hunk 2]
```diff
@@ -2869,16 +2869,18 @@ Basic parameters
      - ``MIN_FIELD``
      - [tablefield: numeric]
      - Field with the minimum value of the range for the class.
+       Use -inf to capture the lowest possible value.
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/rasteranalysis.rst:2872]:
```suggestion
       Use ``-inf`` to capture the lowest possible value.
```

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst [hunk 3]
```diff
@@ -2869,16 +2869,18 @@ Basic parameters
...
      - [tablefield: numeric]
      - Field with the maximum value of the range for the class.
+       Use inf to capture the highest possible value.
    * - **Output value field**
      - ``VALUE_FIELD``
      - [tablefield: numeric]
      - Field with the value that will be assigned to the pixels that
        fall in the class (between the corresponding min and max
-       values).
+       values). Use nan to set the value of the range to no data.
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/rasteranalysis.rst:2883]:
```suggestion
       values). Use ``nan`` to set the value of the range to no data. 
```

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst [hunk 4]
```diff
@@ -3004,6 +3006,8 @@ Basic parameters
      - A 3-columns table to fill with the values to set the boundaries
        of each class (``Minimum`` and ``Maximum``) and the new
        ``Value`` to assign to the band values that fall in the class.
+       The value -inf can be used as minimum, inf as maximum,
+       and nan can be used to set the output value to no data.
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/rasteranalysis.rst:3010]:
```suggestion
       The value ``-inf`` can be used as minimum, ``inf`` as maximum,
       and ``nan`` can be used to set the output value to no data.
```

**Review by DelazJ — COMMENTED** [review]
Just a formatting suggestion to emphasize these values

**Comment by baswein** [comment]
Sometimes the docs use NoData or nodata or no data. I used no data because that was what was already used in this particular algorithm. Any opinion? Should we harmonize this?

**Pull request merged**